torchtune/dev/rl/workers/trainers/pytorch_actor.py

```diff
@@ -739,10 +739,10 @@ def _prepare_trajectory(self, raw_trajectory):
         seq_lens = raw_trajectory.seq_lens
         answers = raw_trajectory.answers
         policy_version = raw_trajectory.policy_version
-        rewards = raw_trajectory.rewards
+        # rewards = raw_trajectory.rewards
         advantages = raw_trajectory.advantages
-        successes = raw_trajectory.successes
-        reward_metadata = raw_trajectory.reward_metadata
+        # successes = raw_trajectory.successes
+        # reward_metadata = raw_trajectory.reward_metadata
```

Comment:
Hmmm, this one is no longer needed as we now log it. I wonder if having a pointer to it in the trajectory is beneficial though